interface-definitions/include/bgp/protocol-common-config.xml.i

```diff
@@ -1249,6 +1249,12 @@
         <valueless/>
       </properties>
     </leafNode>
+    <leafNode name="disable-ebgp-connected-route-check">
+      <properties>
+        <help>Disable checking if nexthop is connected on ebgp session</help>
```

Comment:
```
Disable check to see if eBGP peer address is a connected route
```
As we  have this per peer
```
   disable-connected-check
                        Disable check to see if eBGP peer address is a connected route

```

Reply:
that description is wrong, the correct should be : 
```
vrf-test(config-router)# neighbor 1.1.1.1 disable-
  disable-connected-check        one-hop away EBGP peer using loopback address
```
https://docs.frrouting.org/en/latest/bgp.html#clicmd-neighbor-PEER-disable-connected-check

Reply:
At least it should be `eBGP` but not `ebgp`

Reply:
Done!

Reply:
Can you fix the description of the neighbor command in a subsequent commit of this PR please?

Reply:
I've added a description more clear for `disable-connected-check`